webapi/src/main/scala/org/knora/webapi/slice/admin/api/Codecs.scala

```diff
@@ -9,28 +9,15 @@ import sttp.tapir.Codec
 import sttp.tapir.CodecFormat
 import zio.json.JsonCodec
 
-import dsp.valueobjects.Group.GroupDescriptions
-import dsp.valueobjects.Group.GroupName
-import dsp.valueobjects.Group.GroupSelfJoin
-import dsp.valueobjects.Group.GroupStatus
 import dsp.valueobjects.V2
 import org.knora.webapi.slice.admin.api.model.MaintenanceRequests.AssetId
-import org.knora.webapi.slice.admin.domain.model.Email
-import org.knora.webapi.slice.admin.domain.model.FamilyName
-import org.knora.webapi.slice.admin.domain.model.GivenName
-import org.knora.webapi.slice.admin.domain.model.GroupIri
 import org.knora.webapi.slice.admin.domain.model.KnoraProject.*
 import org.knora.webapi.slice.admin.domain.model.ListProperties.Comments
 import org.knora.webapi.slice.admin.domain.model.ListProperties.Labels
 import org.knora.webapi.slice.admin.domain.model.ListProperties.ListIri
 import org.knora.webapi.slice.admin.domain.model.ListProperties.ListName
 import org.knora.webapi.slice.admin.domain.model.ListProperties.Position
-import org.knora.webapi.slice.admin.domain.model.Password
-import org.knora.webapi.slice.admin.domain.model.RestrictedViewSize
-import org.knora.webapi.slice.admin.domain.model.SystemAdmin
-import org.knora.webapi.slice.admin.domain.model.UserIri
-import org.knora.webapi.slice.admin.domain.model.UserStatus
-import org.knora.webapi.slice.admin.domain.model.Username
+import org.knora.webapi.slice.admin.domain.model.*
```

Comment:
Removing explicit imports? No, I did that on purpose, would you like me to revert it? In regards to the order, that was scalafix invoked by metal's _organize imports_ functionality.

Reply:
Don't worry, no need to revert. The issue is that intellij doesn't support scalafix - we used to have scalafix enabled in the CI formatting check, but that caused constantly failing CI runs, which got annoying.  
This is really not important for now, but in the long run it will probably be nice to ensure that everyone's setup results in roughly the same imports, so that we don't have unnecessary git diffs to ignore in our reviews.